src/StructuralIdentifiability.jl

```diff
@@ -130,14 +131,14 @@ function assess_identifiability(ode::ODE{P}, funcs_to_check::Array{<: RingElem,
     return result
 end
 
-function assess_identifiability(ode::ModelingToolkit.ODESystem, inputs, funcs_to_check, p::Float64=0.99)
-    diff_eqs = equations(ode)
-    params = ModelingToolkit.parameters(ode)
-    state_vars = ModelingToolkit.states(ode)
-    y_functions = [each.lhs for each in ModelingToolkit.observed(ode)]
-    output_eqs = ModelingToolkit.observed(ode) 
+function assess_identifiability(de::ModelingToolkit.ODESystem, inputs, funcs_to_check, p::Float64=0.99)
+    diff_eqs = equations(de)
+    params = ModelingToolkit.parameters(de)
+    state_vars = ModelingToolkit.states(de)
+    y_functions = [each.lhs for each in ModelingToolkit.observed(de)]
+    output_eqs = ModelingToolkit.observed(de)
     ode, syms, gens_ = PreprocessODE(diff_eqs, output_eqs, state_vars, y_functions, inputs, params)
-    if length(funcs_to_check)>0
+    if length(funcs_to_check) > 0
         funcs_to_check = [substitute(x, syms .=> gens_) for x in funcs_to_check]
```

Comment:
I believe that this should also be `eval_at_nemo`

Reply:
could you please clarify?

Reply:
The `func_to_check` may also be some arbitrary expressions of parameters (including fractions and whatnot), so I think it would make sense to evaluate them by the same code as the RHS of the equations. Does this make more sense now?